evm/src/main/java/org/hyperledger/besu/evm/tracing/OperationTracer.java

```diff
@@ -69,7 +69,7 @@ default void traceAccountCreationResult(
    *
    * @param transaction the transaction which will be processed
    */
-  default void traceStartTransaction(final Transaction transaction) {}
+  default void traceStartTransaction(@SuppressWarnings("unused") final Transaction transaction) {}
```

Comment:
I think the `@SuppressWarnings` annotations are not strictly needed here. The CI pipeline should be okay with it not being used, because before it passed for `traceStartTransaction` as well.

Reply:
To be frank I didn't understand why it complained about unused parameters in `traceStartTransaction` but not in `traceEndTransaction`...